### ISSUE TITLE
Accessibility: organization page has an empty heading (with no content) on the page

### ISSUE THREAD

**Issue by Dawnzar** [CityofOttawa/ckanext-ottawa#8]
WCAG 2.0 - Success Criteria - 1.3.1 Level (A) violation
1. Go to: http://data.ottawa.ca/organization
2. View source
3. Please observe the (h?) tag which needs to be removed.

Resolution: Remove or name tag.

![h](https://cloud.githubusercontent.com/assets/5271253/8034561/d4d06e6e-0db6-11e5-8752-12b0727ce573.JPG)

**Comment by deniszgonjanin** [comment]
@Dawnzar ready for review

**Comment by Dawnzar** [comment]
Retested and resolved.
